src/rust/engine/fs/store/src/remote_tests.rs

```diff
@@ -308,7 +308,11 @@ async fn list_missing_digests_error() {
   );
 }
 
-#[tokio::test]
+// TODO(tonic): Ignored this test because, while `ByteStore` is configured with both endpoints,
```

Comment:
Tonic did not seem to round robin all the time. This test technically relies on an implementation detail of grpcio.